nexus/Freqlog/Freqlog.py

```diff
@@ -19,19 +19,16 @@
 class Freqlog:
 
     def _on_key(self, key: vinput.KeyboardEvent) -> None:
-        type = ActionType.PRESS
+        kind = ActionType.PRESS
         if not key.pressed:
-            type = ActionType.RELEASE
+            kind = ActionType.RELEASE
         if key.keychar == '' or key.keychar == '\0':
             return
-        self.q.put((type, key.keychar.lower(), key.modifiers, datetime.now()))
+        self.q.put((kind, key.keychar.lower(), key.modifiers, datetime.now()))
 
     def _on_mouse_button(self, button: vinput.MouseButtonEvent) -> None:
         """Store PRESS, key and current time in queue"""
-        mods = vinput.KeyboardModifiers()
-        for attr in Defaults.MODIFIER_NAMES:
-            setattr(self.modifier_keys, attr, True)
-        self.q.put((ActionType.PRESS, '', mods, datetime.now()))
+        self.q.put((ActionType.PRESS, button, None, datetime.now()))
```

Comment:
This is now sending `None` instead of the modifier keys, you should correctly reflect the modifier keys.

Reply:
aren't the modifier keys ignored now? it shouldn't matter

Reply:
Pass them anyway, for debugging purposes and future extensibility.

Reply:
Nevermind, there apparently are no modifiers collected because the listeners for keyboard (modifiers) and mouse are separate.